### PR TITLE
XW-1302 | Add firstRenderTime measurement

### PR DESCRIPTION
**Pull request by rogatty**
## Links

* https://wikia-inc.atlassian.net/browse/XW-1302
* https://davidwalsh.name/css-animation-callback

## Description

We want to know when users actually see the content and it turns out that `domContentLoaded` isn't the correct event to measure that.

Instead, let's use an empty element with CSS transition applied and measure when the transition is finished. I've run some tests on http://webpagetest.org and it seems that this time is the correct one to measure.

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: server/app/views/_partials/first-render-time.hbs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{{! @todo (XW-1374) this should be an Ember addon }}
+{{! Idea from https://davidwalsh.name/css-animation-callback }}
+<span class="render-time-test"></span>
+<style type="text/css">
+	.render-time-test {
+		transition-property: opacity;
+		transition-duration: .000001s;
```
Comment (vforge) [inline server/app/views/_partials/first-render-time.hbs]: It's `0.001s` minimal, so you can use just `1ms`

**Comment by vforge** [comment]
LGTM, one minor change is suggested though.

**Comment by vforge** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Pull request merged**